run/events-pubsub/README.md

```diff
@@ -0,0 +1,47 @@
+# Events for Cloud Run – Pub/Sub tutorial
```

Comment:
Might we need different CODEOWNERS for events?

Reply:
Maybe. How about we add CODEOWNERS after we have more than a couple samples.